src/lighteval/models/model_loader.py

```diff
@@ -94,6 +97,9 @@ def load_model(  # noqa: C901
     if isinstance(config, DummyModelConfig):
         return load_dummy_model(config=config, env_config=env_config)
 
+    if isinstance(config, VLLMModelConfig):
```

Comment:
If vllm is not accelerate compatible, we might want to separate `load_model_with_default` and `load_model_with_accelerate`

Reply:
can be confusing otherwise

Reply:
I think it's only the name that is confusing, because what is `load_model_with_default` ?